### DISCUSSION TITLE
Hving Issue when pausing the video

### DISCUSSION THREAD

**Discussion by SaiGaneshan2** [FreeTubeApp/FreeTube#6102] [General]
Hi, I use FreeTube as my main youtube , when i pause the video to take notes as student , the paused icon is soo big that , it is difficult for me to copy notes or read content when the video is paused.

I want the paused icon to be smaller on the video screen.
![freetube](https://github.com/user-attachments/assets/ce4a2bda-6e32-4507-b5f1-a2b465f6b18d)

**Comment by absidue** [comment]
Duplicate of #5951
